connectors/v2/deezer.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#page_sidebar';
+
+Connector.getArtist = function() {
+	var artists = $.map($('.player-track-artist').children(), function(artist) {
+		return $.text(artist);
+	});
+	return artists.join(', ');
```

Comment:
That's an example how an artists element looks:
```html
<h3 class="player-track-artist heading-2-sub ellipsis">
    <!-- react-text: 1666 -->from<!-- /react-text -->
    <!-- react-text: 1667 -->&nbsp;<!-- /react-text -->
    <!-- react-text: 1668 --><!-- /react-text -->
    <span class="player-track-link" data-action="link" data-href="/artist/282118">Major Lazer</span>
    <!-- react-text: 1670 -->, <!-- /react-text -->
    <span class="player-track-link" data-action="link" data-href="/artist/288166">Justin Bieber</span>
    <!-- react-text: 1672 -->, <!-- /react-text -->
    <span class="player-track-link" data-action="link" data-href="/artist/4333887">MO</span>
</h3>
```
If `player-track-artist` is used as selector, the actual artist is 'from Major Lazer, Justin Bieber, MO'. Of course, artists can be extracted by splitting the whole string by non-breaking space and taking the second part of the string. But I think it's not a good way. So I decided to get children of `.player-track-artist`, convert them to an array of strings and join by comma.